components/SkipLogin/SkipLoginButton.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from "./SkipLoginButton.module.css";
+
+function SkipLoginButton() {
+  function handleClick(e) {
+    e.preventDefault();
+    alert("This feature will be available in a later version of the app.");
+  }
+
+  return (
+    <a href="#" onClick={handleClick}>
```

Comment:
It's rather uncommon to have an `<a>`-Tag have a onClick property.
You could give it to the button and remove the `<a>`-Tag altogether till you add the feature